.buildkite/pipeline.trigger.integration.tests.sh

```diff
@@ -49,6 +49,22 @@ for test in ${CHECK_PACKAGES_TESTS[@]}; do
     fi
 done
 
+pushd test/packages/false_positives > /dev/null
+for package in $(find . -maxdepth 1 -mindepth 1 -type d) ; do
+    package_name=$(basename ${package})
+    echo "      - label: \":go: Running integration test: ${package_name}\""
```

Comment:
Should it be indicated here that is from `false positive`? 

```suggestion
    echo "      - label: \":go: Running integration test (false positive): ${package_name}\""
```